### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: protocol/rtmp/fuzz/fuzz_targets/read_chunks.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#![no_main]
```
Comment (xansec) [inline protocol/rtmp/fuzz/fuzz_targets/read_chunks.rs:1]: Does this fuzz xiu or rtmp?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: protocol/rtmp/fuzz/fuzz_targets/read_chunks.rs (hunk 1 above)
Comment (evanrichter) [reply protocol/rtmp/fuzz/fuzz_targets/read_chunks.rs:1]: this is a utility crate that is first-party code to this repo. they don't use vendor sources. If you go to [crates.io](https://crates.io/crates/rtmp) you'll see that they do publish this crate standalone as a library for others to use. But the upstream repo is here.

**Review by xansec — APPROVED** [review]
Gotcha. This is ok with me. LGTM!

**Pull request merged**